<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UfrmMilitancyUpdated</class>
 <widget class="QMainWindow" name="UfrmMilitancyUpdated">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1322</width>
    <height>691</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Ingreso militancia</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/militant_32.png</normaloff>Resources/militant_32.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelMinorYearsMilitancy">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Estudiantes cuyo año de ingreso es distinto al especificado</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelSameYearMilitancy">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Estudiantes cuyo año de ingreso es igual al especificado</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableStudentsMinorMilitancy">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>401</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>CI</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombres </string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellidos</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ingreso
UJC</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableStudentsSameMilitancy">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>60</y>
      <width>401</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>CI</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombres </string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellidos</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ingreso
UJC</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="labelTotalMinorMilitancy">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>570</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Total: </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTotalElderMilitancy">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>570</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Total: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushAcceptMilitancy">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>620</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Aceptar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/accept_32.png</normaloff>Resources/accept_32.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
